server/models/Group.js

```diff
@@ -250,6 +250,15 @@ export default function(Sequelize, DataTypes) {
     },
 
     instanceMethods: {
+      getUsersForViewer(viewer) {
+        const promises = [this.getUsers()];
```

Comment:
.getUsers() isn't going to be reliable for our setup, it's going to miss users/roles. Because we removed the unique index on UserGroup, which assumed a unique user and group combo. Our unique index is on user, group and role combo.